backend/routes/posts.ts

```
@@ -0,0 +1,192 @@
+        typeof item.createdAt === 'string' &&
+        typeof item.replyCount === 'number' && item.replyCount >= 0
+        // parentId no longer exists on Post
+    );
+}
+
+/**
+ * @route   POST /api/posts/createPost
+ * @desc    Creates a new top-level post (story)
+ * @access  Authenticated
+ */
+router.post<{}, any, { postTree: PostCreationRequest }>(
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)